### ISSUE TITLE
Error: unable to get local issuer certificate error while creating serverless project

### ISSUE THREAD

**Issue by ashwineev** [serverless/serverless#11208]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [X] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

I am trying to create serverless project 

While creating project I get below error 

Error: unable to get local issuer certificate
    at TLSSocket.onConnectSecure (node:_tls_wrap:1532:34)
    at TLSSocket.emit (node:events:527:28)
    at TLSSocket.emit (node:domain:475:12)
    at TLSSocket._finishInit (node:_tls_wrap:946:8)
    at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:727:12)

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

serverless

### Command output

```shell
Creating a new serverless project

? What do you want to make? AWS - Python - HTTP API
? What do you want to call this project? serve1

✔ Project successfully created in serve1 folder
Environment: win32, node 16.15.1, framework 3.19.0, plugin 6.2.2, SDK 4.3.2
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Error: unable to get local issuer certificate
    at TLSSocket.onConnectSecure (node:_tls_wrap:1532:34)
    at TLSSocket.emit (node:events:527:28)
    at TLSSocket.emit (node:domain:475:12)
    at TLSSocket._finishInit (node:_tls_wrap:946:8)
    at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:727:12)
```

### Environment information

```shell
Framework Core: 3.19.0
Plugin: 6.2.2
SDK: 4.3.2
```

**Comment by pgrzesik** [comment]
Hello @ashwineev - are you using some local proxy by any chance?

**Comment by ashwineev** [comment]
> Hello @ashwineev - are you using some local proxy by any chance?
@pgrzesik Could you please let me know how to check this?

**Comment by ashwineev** [comment]
@pgrzesik  yes. I am using corporate system , but even if I disable the VPN network and run the command I get the same error. Could you please help.

**Comment by pgrzesik** [comment]
I'm not sure what might work in your case but you can find some ideas here: https://github.com/serverless/serverless/issues/3256 and in a few other related issues. I'm going to close this as it's not a bug with the Framework
